app/src/androidTest/kotlin/dev/msfjarvis/aps/db/StoreDaoTest.kt

```diff
@@ -46,24 +46,24 @@ class StoreDaoTest {
   @Test
   @Throws(Exception::class)
   fun testInsertStore() {
-    val store = StoreEntity(name = "store", external = false, initialized = true)
+    val store = StoreEntity(name = "store", external = false, initialized = true, isGitStore = false, uri = null)
     storeDao.insertStore(store)
     val byName = storeDao.getStoreByName("store").asLiveData().blockingObserve()
-    assertThat(byName.get(0).name, equalTo(store.name))
+    assertThat(byName?.get(0)?.name, equalTo(store.name))
   }
 
   @Test
   @Throws(Exception::class)
   fun testInsertMultipleStores() {
     val storeEntries = arrayListOf<StoreEntity>()
     for (i in 0 until 7) {
-      storeEntries.add(StoreEntity(name = "store$i", external = false, initialized = false))
+      storeEntries.add(StoreEntity(name = "store$i", external = false, initialized = false, isGitStore = false, uri = null))
```

Comment:
Do we need to pass all parameters by name? Seems excessively verbose

Reply:
That was an android studio suggestion, will revert it.